lab_3_ann_retriever/main.py

```diff
@@ -104,6 +120,21 @@ def tokenize(self, text: str) -> list[str] | None:
 
         In case of corrupt input arguments, None is returned.
         """
+        if not isinstance(text, str) or not text:
+            return None
+        self.text = text
+        new_text = ''
```

Comment:
Strange implementation, try to simplify it

Reply:
I did the similar in the previous laboratory work

Reply:
Be better today than you were yesterday 
![image](https://github.com/user-attachments/assets/9f77b938-9935-4564-9026-c46e12952094)

Reply:
Done with replace()